### PR TITLE
Switch to mocha for unit tests

### PR DESCRIPTION
**Pull request by Jikstra**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -136,12 +139,14 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "glob-watcher": "^5.0.5",
     "hallmark": "^4.0.0",
+    "mocha": "^9.1.4",
     "prettier": "2.2.1",
     "sass": "^1.26.5",
     "tape": "^4.12.0",
```
Comment (dotlambda) [inline package.json:145]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 - Update inApp help (3.1.2022)
 - Making the minimal them even more zen
 - Drastically improve performance of MessageList
+- Fix from tape to mocha for unit tests
```
Comment (dotlambda) [inline CHANGELOG.md:11]:
```suggestion
- Switch from tape to mocha for unit tests
```

**Comment by Jikstra** [comment]
Hey @dotlambda good to see you around again :) Thanks for the fixes!

File: src/shared/localize.ts [hunk 3]
```diff
@@ -5,7 +5,6 @@ export interface LocaleData {
   locale: string
   messages: {
     [key: string]: {
-      message: string
       [key: string]: string
```
Comment (Simon-Laux) [inline src/shared/localize.ts:8]:
```suggestion
      message?: string,
      one?: string,
      other?: string,
```

**Pull request merged**